src/inc/msquic_winuser.h

```diff
@@ -116,6 +116,7 @@
 #define QUIC_STATUS_ALPN_NEG_FAILURE        ERROR_QUIC_ALPN_NEG_FAILURE                     // 0x80410007
 #define QUIC_STATUS_STREAM_LIMIT_REACHED    ERROR_QUIC_STREAM_LIMIT_REACHED                 // 0x80410008
 #define QUIC_STATUS_ALPN_IN_USE             ERROR_QUIC_ALPN_IN_USE                          // 0x80410009
+#define QUIC_STATUS_ADDRESS_NOT_AVAILABLE   HRESULT_FROM_WIN32(WSAENETUNREACH)              // 0x80072743
```

Comment:
We already have `#define QUIC_STATUS_INVALID_ADDRESS         HRESULT_FROM_WIN32(WSAEADDRNOTAVAIL)`. I'm not so sure about this change.

Reply:
Is the naming QUIC_STATUS_ADDRESS_NOT_AVAILABLE that bothers you or WSAENETUNREACH as windows error code ?

Reply:
Yes, I don't like that the names don't match. I don't know which name should be preferred, because I am not sure what exactly they are being used for. I'd prefer if there was a QUIC test that explained/validated them.

Reply:
Sorry I didn't find a way to programmatically deactivate/reactivate IPv6. If you had any idea on how to achieve it I would be thankfull

Reply:
You can do via registry AFAIK but that works only for external targets and it requires admin access e.g. it is not good candidate IMHO for automated tests